ui/src/markup.tsx

```diff
@@ -59,22 +62,26 @@ interface State {
    * :value "<div/>"
    **/
   content: S
+  /** 
+   * True if title and padding should be removed. Defaults to False.
```

Comment:
```suggestion
   * True if outer spacing should be removed. Defaults to False.
```

Don't forget to run `make generate` after fixing this.